### DISCUSSION TITLE
Batterie nachts sogar teilweise geladen nutzen

### DISCUSSION THREAD

**Discussion by hoschiking** [hoylabs/OpenDTU-OnBattery#1303] [Q&A] [answered]
Hallo,
kann mich jemand aufklären, welchen Sinn dieser neue Schalter hat?
Ich finde auch keine Beschreibung hierfür im Projekt.
Mir fällt auch kein Szenarion ein, bei dem der Schalter einen Sinn ergeben sollte.
Vielleicht übersehe ich aber auch etwas.

**Comment by schlimmchen** [comment]
> Vielleicht übersehe ich aber auch etwas.

Ja, z.B. [die Dokumentation](https://opendtu-onbattery.net/firmware/configuration/dpl/#use-battery-at-night-even-if-only-partially-charged).

Der Schalter ist nicht neu. Er hat vor Monaten schon die Einstellung "Battery Discharge Strategy" (oder so ähnlich) abgelöst.

**Reply by hoschiking** [reply]
Danke für den Hinweis, habe es echt übersehen, wenn auch ich den Sinn des Schalters immer noch nicht verstehe. Die Startschwelle für die Entladung wird doch entweder über den soc oder die Start und Stop Spannung definiert, wie in meinem Fall. Der soc wird ignoriert, weil in der Praxis nicht wirklich brauchbar, wenn das BMS mit der soc Berechnung weg driftet. Soweit ich beobachtet habe, wird die Batterie auch dann entladen, wenn dieser Schalter aus ist und die Start Spannung erreicht wurde. Dies sollte auch so sein, weil wenn der Schalter soc ignorieren aktiviert ist, der soc nicht berücksichtigt werden darf. Um die Frage etwas konkreter zu stellen: welches Verhalten des dpl stellt sich beim Aktivieren bzw deaktivieren des Schalters ein?

**Reply by schlimmchen** [reply, marked as answer]
Es wird Nacht und die Batterie hat die Start-Schwelle aber (trotz Ladens) nicht erreicht => Batterie wird dennoch entladen. Wenn der Schalter aus ist, wird die Batterie nicht entladen, sondern am nächsten Tag weitergeladen (wenn denn Sonne da ist).

Wenn deine Start-Spannung niedrig ist, hat der Schalter effektiv keinen Sinn. So ist es bei mir auch (51V Stop-Spannung, 52V Start-Spannung).

Mit dem SoC hat der Schalter nichts zu tun, er bezieht sich nur auf die Schwellen, egal ob die in SoC[%] oder Spannung[V] ausgedrückt sind.

Der Schalter/Modus ist in der Tat recht schwer zu begreifen. Ich hatte aber gehofft, er sei inzwischen hinreichend einleuchtend und zusätzlich auch noch erklärt. Was fehlt dir in der Doku, damit du das verstanden hättest? Ich versuche das gerne noch zu verbessern.

**Reply by hoschiking** [reply]
Der erste Absatz ist genau die Erklärung, die bisher gefehlt hat. Danke 👍 
Das ergibt auch Sinn. 
Dann ist die Benennung des Schalters und die Erklärung etwas unscharf und beschreibt in meinen Augen nicht dass Verhalten. 
Wenn du es in der Doku analog zum ersten Absatz deiner Erklärung beschreiben würdest, würde ich es sogar verstehen ;-)
Vorschlag für den Schalter: 
Batterie entladen sogar wenn Startschwelle nicht erreicht.
Nur so als Idee

**Reply by hoschiking** [reply]
Und dann noch den Schalter räumlich dahin platzieren, wo auch die Schwellen definiert werden, dann ist es perfekt. Dann weiß man, worauf sich der Schalter bezieht.
